paddle/fluid/framework/executor.cc

```diff
@@ -350,12 +350,9 @@ void Executor::RunPreparedContext(ExecutorPrepareContext* ctx, Scope* scope,
       }
     }
   }
-  platform::DeviceContextPool::Instance().Get(place_)->Wait();
+  // platform::DeviceContextPool::Instance().Get(place_)->Wait();
```

Comment:
Don't need wait?

Reply:
Why delete `DropKids()`?

Reply:
Done. (It used to crash, seems to be fixed now)